src/openforms/registrations/tasks.py

```diff
@@ -162,31 +192,40 @@ def register_submission(submission_id: int) -> None:
 
     if not plugin.is_enabled:
         logger.debug("Plugin '%s' is not enabled", backend)
-        e = RegistrationFailed("Registration plugin is not enabled")
+        exc = RegistrationFailed("Registration plugin is not enabled")
         submission.save_registration_status(
-            RegistrationStatuses.failed, {"traceback": str(e)}
+            RegistrationStatuses.failed, {"traceback": str(exc)}
         )
-        logevent.registration_failure(submission, e)
-        raise e
+        if event == PostSubmissionEvents.on_retry:
+            raise exc
+        return
 
     logger.debug("De-serializing the plugin configuration options")
     options_serializer = plugin.configuration_options(data=backend_config.options)
+
     try:
         options_serializer.is_valid(raise_exception=True)
-    except Exception as e:
+    except ValidationError as exc:
+        logevent.registration_failure(submission, exc, plugin)
+        logger.warning(
+            "Registration using plugin '%r' for submission '%s' failed",
+            plugin,
+            submission,
+        )
```

Comment:
```suggestion
        logger.warning(
            "Registration using plugin '%r' for submission '%s' failed",
            plugin,
            submission,
            exc_info=exc,
        )
```
when an exception is available, pass it along so we can see more details in Sentry :)